### DISCUSSION TITLE
roadmap

### DISCUSSION THREAD

**Discussion by 91nsonline** [pkgxdev/pkgx#943] [Help]
Where can I read about the detailed roadmap?

**Comment by codewithneeraj** [comment]
Hi @91nsonline,
You can checkout below link to see the Roadmap.
Roadmap: https://tea.xyz/learn/roadmap

**Reply by minhdataaa** [reply]
Thank you, I also looking for to check again !

**Comment by chris-setra** [comment]
https://tea.xyz/learn/roadmap

**Comment by anhtuan111111** [comment]
you can see here. Surely it will help you
https://tea.xyz/learn/roadmap

**Comment by tandotendo** [comment]
see this Roadmap $TEA : https://tea.xyz/learn/roadmap
